docker/images/pinot/etc/jmx_prometheus_javaagent/configs/controller.yml

```diff
@@ -48,6 +48,21 @@ rules:
     table: "$2$4"
     tableType: "$5"
     taskType: "$6"
+  # Special handling for timers like cronSchedulerJobExecutionTimeMs and tableRebalanceExecutionTimeMs
```

Comment:
Doesn't look like it, but any way we don't have to add these? We want to keep this rule set as minimal as possible.

Reply:
I tried going through the other existing options but they don't seem to work for these. any other recommendations?

Reply:
I don't know if this is better, but I can probably combine both into a single regex, and just use `taskType` for both:

```
  # Special handling for timers like cronSchedulerJobExecutionTimeMs and tableRebalanceExecutionTimeMs
- pattern: "\"org\\.apache\\.pinot\\.common\\.metrics\"<type=\"ControllerMetrics\", name=\"pinot\\.controller\\.(([^.]+)\\.)?([^.]*)\\.(\\w+)\\.(tableRebalanceExecutionTimeMs|cronSchedulerJobExecutionTimeMs)\"><>(\\w+)"
  name: "pinot_controller_$5_$6"
  cache: true
  labels:
    database: "$2"
    table: "$1$3"
    taskType: "$4"
```

Reply:
We can do that but I think that'll not be idiomatic for `tableRebalanceExecutionTimeMs` (`taskType=DONE` doesn't make much sense). I think we can stick to two regexps. cc @Jackie-Jiang

Reply:
actually `operationStatus` may not make much sense either since for the `cronSchedulerJobExecutionTimeMs` the argument is actually supposed to be the task name, e.g. `taskType="SegmentImportTask"`